modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
 . Save the `.bashrc` file.
 . Restart your terminal.
 
+== Preparing to Install Instio
+
+When you install Instio in your environment, you must configure Instio settings based on the Instio version that you install.
+
+=== Installing Instio 1.4x
+
+To install and configure Instio 1.4x for Anypoint Service Mesh:
+
+. Download and install Instio using the Instio documentation.
```

Comment:
```suggestion
. Download and install Istio using the Istio documentation.
```

Reply:
This topic is titled PREPARING to install, not installing. I think you need to retitle the container topic....

Reply:
Done! Thanks!